docs/user-guide/calendar/README.md

```diff
@@ -2,84 +2,124 @@
+> Work packages can also expand and retract in width depending on how many non-working days are spanned (for example, a 3-day task starting on Thursday and ending on Monday will spread across 5 calendar days; dragging that same work package so that it starts on a Tuesday and ends on a Thursday means that it will spread across 3 calendar days. In both cases, the duration remains 3 days.
 
+### Filtering
 
+You can filter the work packages displayed in a calendar by adding any number of [filters](../work-packages/work-package-table-configuration/#filter-work-packages "Learn more about work package filters"), e.g. *Author*, *Due Date*, *Priority*, *Assignee*, etc. This allows you to create a custom calendar showing only relevant work packages, either for yourself (as a private calendar) or for your team (as a public calendar visible to all project members).
 
-## Filters in calendar
+Click the **Filter** button on the top right above the calendar to open the filter details.To add another filter criteria, select the drop down next to **+ Add filter** and choose a filter criteria.The number of different filter criteria is displayed on the filter button.
 
-You can filter the work packages in the calendar view by adding any number of filters, e.g. *Author*, *Due Date*, *Priority*, *Assignee*, etc.
+![Calendar filters](calendarFilters.png)
 
-Click the **Filter** button on the top right above the calendar to open the filter details.
+> **Note:** OpenProject also offers a module called Team planner that allows you to view modules in a calendar by assignee. [Learn more about Team planner](../team-planner "Documentation about Team planner").
 
-To add another filter criteria, select the drop down next to **+ Add filter** and choose a filter criteria.
+### Including other projects
 
-The number of different filter criteria is displayed on the filter button.
+Much like with work package tables, it is possible to display work packages from more other projects. Use the **Include projects** dialog, where you can select/unselect from a list of projects and sub-projects. To automatically include all sub-projects for each project you chose to select, check the **Include all sub-projects** box at the bottom of the dialog.
 
-![filter-calendar](filter-calendar.png)
+### Embedding a calendar in the project overview
 
 You can also display the calendar on the dashboard in the [project overview](../project-overview/#calendar-widget).
 
-## Zen mode
+### Zen mode
+
+You can display the calendar full screen mode by hiding the navigation bar  and sidebar to work free of distractions. Click the **Activate zen mode** button on the top of the calendar. Press the escape key (**Esc**) to exist Zen mode.
+
+## Subscribing to a calendar
+
+OpenProject allows you to subscribe to and access any of your calendars using an external client that supports the .ics format.
```

Comment:
Hi @psatyal,

We are always talking about iCalendar format not .ics, it is true that the file generated is ICS but the iCalendar is the URL option to keep it updated itself not the downloaded file in ics. I would talk about iCalendar here.

Reply:
Ahh good point. I've changed it to:

> OpenProject allows you to subscribe to and access any of your calendars using an external client that supports the iCalendar format (such as Thunderbird, Open-Xchange, Apple Calendar or Google Calendar).